app/_src/gateway/install/docker/build-custom-images.md

```diff
@@ -142,8 +143,14 @@ RUN set -ex; \
 {% endcapture %}
 {{ dockerfile | indent }}
 
-1. Build your image with `docker build --no-cache -t kong-your-tag .`
+1. Build your image:
+```bash
+docker build --no-cache -t kong-image .
+```
 
-1. Test that the image built correctly with `docker run -it kong-your-tag kong version`
+1. Test that the image built correctly:
+```
+docker run -it kong-image kong version
```

Comment:
I don't know your policy on this, but it seems good practice to discard the created one-off container after execution, since it will probably never be required again.

```suggestion
docker run -it --rm kong-image kong version
```